store/src/main/scala/geotrellis/store/Implicits.scala

```diff
@@ -16,12 +16,16 @@
 
 package geotrellis.store
 
-import geotrellis.raster.CellGrid
-import geotrellis.layer._
-import geotrellis.util._
-import java.time.Instant
-
+import geotrellis.layer.{SpaceTimeKey, SpatialKey, TileLayerMetadata}
 
 object Implicits extends Implicits
 
-trait Implicits extends avro.codecs.Implicits with json.Implicits
\ No newline at end of file
+trait Implicits extends avro.codecs.Implicits with json.Implicits {
+  implicit class AttributeStoreOps(attributeStore: AttributeStore) {
+    private [store] def readTileLayerMetadataErased(layerId: LayerId): TileLayerMetadata[_] = {
```

Comment:
An arguable descision, but can be a convenientl internal API for GeoTrellisRasterSources